test/EFCore.MySql.UpstreamFunctionalTests/MigrationSqlGeneratorMySqlTest.cs

```diff
@@ -514,7 +514,7 @@ INTO PRIMARY_KEY_COLUMN_NAME
 		EXECUTE SQL_EXP_EXECUTE;
 		DEALLOCATE PREPARE SQL_EXP_EXECUTE;
 	END IF;
-END;" + EOL +
+END;".Split('\n')) + EOL +
```

Comment:
To make this resilient to future line ending changes it would be better to call:
```
.Replace("\r", string.Empty).Replace("\n", EOL)
```